README.md

```diff
@@ -28,42 +42,38 @@ There are several ways to install this app onto a workload cluster.
 
 ```yaml
 # values.yaml
-
+global:
+  remoteWrite:
+  - name: "remotewrite" 
+    url: "http://$BASEDOMAIN/write"
+
+prometheus-agent:
+  serviceMonitor:
+    enabled: true
+    relabelings:
+    # Add app label.
+    - sourceLabels:
+      - __meta_kubernetes_pod_label_app_kubernetes_io_name
+      targetLabel: app
+    # Add instance label.
+    - sourceLabels:
+      - __meta_kubernetes_pod_label_app_kubernetes_io_instance
+      targetLabel: instance
+    # Add node label.
+    - sourceLabels:
+      - __meta_kubernetes_pod_node_name
+      targetLabel: node
 ```
 
-### Sample App CR and ConfigMap for the management cluster
+### Conflicts 
 
-If you have access to the Kubernetes API on the management cluster, you could create
-the App CR and ConfigMap directly.
-
-Here is an example that would install the app to
-workload cluster `abc12`:
+To avoid conflicts with other prometheuses deployed to same cluster,
+you can use selectors for `ServiceMonitors` and `PodMonitors`:
 
 ```yaml
-# appCR.yaml
-
+prometheus-agent:
+  serviceMonitorSelector:
+    organization: giantswarm
```

Comment:
Let's wait for feedback on @QuentinBisson 's slack thread to chose the label name.
But I think this documentation should be customer-oriented, so we should show a value that reflects what the customers would set. Like `organization: my-org`.

Reply:
@mcbenjemaa can you change `organization: giantswarm` to `organization: my-org`? Then we can merge